vnext/Microsoft.ReactNative.Cxx/JSValue.natvis

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
+  <Type Name="winrt::Microsoft::ReactNative::JSValue">
+    <DisplayString Condition="m_type == JSValueType::Null">null</DisplayString>
+    <DisplayString Condition="m_type == JSValueType::Object">{m_object}</DisplayString>
+    <DisplayString Condition="m_type == JSValueType::Array">{m_array}</DisplayString>
+    <DisplayString Condition="m_type == JSValueType::String">{m_string}</DisplayString>
+    <DisplayString Condition="m_type == JSValueType::Boolean">{m_bool}</DisplayString>
+    <DisplayString Condition="m_type == JSValueType::Int64">{m_int64} (A={(int8_t)(m_int64>>24),x}, R={(int8_t)(m_int64>>16 &amp; 0xff),x}, G={(int8_t)(m_int64>>8 &amp; 0xff),x}, B={(int8_t)(m_int64 &amp; 0xff),x})</DisplayString>
```

Comment:
I'm not sure we want to assume all Int64s are colors.

Reply:
Not all are, some will be. We don't have a separate color type, so better to show both representations and allow the user to figure out what the value represents than to show a color like black as "4278190080"

Reply:
I don't think we should cater so far to a single use case, at the cost of others. JSValue is conceptually just a JSON value, and can be used in modules, view managers, etc. Colors could also be in different representations (I know some modules that do this already).

Printing out channel values for all ints will sometimes be wrong in the sense that an int might not be a color, but it can be even more deceptive when it is a color in a different representation. The JSValue is conceptually oblivious to what's inside of it.

Imagine if we did this in a JS debugger, where every number that was a method parameter also showed channel values in some representation. Wouldn't that be confusing?

If someone did know that an int corresponded to a color of a certain scheme, they could always view the number as hex, and quickly see channel values for the color.

Reply:
If we had the same flexibility as in .Net and we could determine with a reasonalbe heuristic is is a oclor: 
i.e. if the fieldname is 'color' or 'backgroundColor' and field value is an int64, then show color breakdown would be okay, but I agree with Nick and Vlad here that for every in64 is too much...